.eslintrc.js

```diff
@@ -300,6 +301,7 @@ module.exports = {
         '**/__snapshots__/*.snap',
         'app/scripts/controllers/app-state.test.js',
         'app/scripts/controllers/mmi-controller.test.ts',
+        'app/scripts/controllers/metametrics.test.js',
```

Comment:
question: what lint rules we are trying to ignore here?

Reply:
These are the jest files, i.e. the files that we want linted by the jest rules